config/environments/production.rb

```diff
@@ -89,8 +89,7 @@
   config.log_formatter = ::Logger::Formatter.new
 
   # may be needed for integrating bootstrap with Heroku deployment
-  #config.cache_classes = true
-  #config.serve_static_assets = true
+  # config.cache_classes = true
```

Comment:
could these comments and commented code go off now?  bootstrap seems to work for us on Heroku ...?